west.yml

```diff
@@ -161,7 +161,7 @@ manifest:
       revision: 12019623bbad9eb54fe51066847a7cbd4b4eac57
       path: modules/lib/loramac-node
     - name: lvgl
-      revision: df717ac87a9fd80246cc8df24554475e59bda21f
+      revision: 892820523cfdd5dbf6d57b1fb4e7d0c0b573b94f
```

Comment:
This should be 07eab463f991f9ea1738a671fa81ad1e54f50ffb. 892820523cfdd5dbf6d57b1fb4e7d0c0b573b94f is the commit SHA from that lvgl PR.

Reply:
done